core/src/main/java/org/geysermc/geyser/entity/type/Tickable.java

```diff
@@ -29,5 +29,7 @@
  * Implemented onto anything that should have code ran every Minecraft tick - 50 milliseconds.
  */
 public interface Tickable {
+    default void drawTick() {};
```

Comment:
Spacing

Reply:
Also, please add a Javadoc explaining the difference between drawTick and tick.

Reply:
Fixed in latest commit.